spec/unit/check_game_state_spec.rb

```diff
@@ -11,46 +11,46 @@
 
   it 'can show when player one wins from a row' do
     game_state_gateway.game_state = [1, 1, 1, 0, 0, 0, 0, 0, 0]
-    expect(check_game_state.execute).to eq('Player one wins')
+    expect(check_game_state.execute).to eq(1)
```

Comment:
I think you could return symbols to improve readability

Reply:
Possibly so. I feel like this leaves it open for when the delivery method is decided upon (e.g. This could hypothetically be used if the delivery method was a grid of LED lights)- I'm not 100% sure if symbols do that so I'd be curious to know if they do

Reply:
I think we'll mark this as something to consider for a future pull request, for the current scope it can be considered an acceptable solution